solr/solrj/src/java/org/apache/solr/client/solrj/impl/ConcurrentUpdateHttp2SolrClient.java

```diff
@@ -454,7 +457,7 @@ public NamedList<Object> request(final SolrRequest<?> request, String collection
             } else {
               long currentStallTime =
                   TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - lastStallTime);
-              if (currentStallTime > stallTime) {
+              if (currentStallTime > stallTimeMillis) {
```

Comment:
And this will go away eventually as setPollQueueTime is deprecated :-). We could make the field volatile but as long as we don't use it within Solr anywhere and it's deprecated, I think it's fine.